common/data/src/main/java/org/thingsboard/server/common/data/tenant/profile/DefaultTenantProfileConfiguration.java

```diff
@@ -190,10 +190,16 @@ public class DefaultTenantProfileConfiguration implements TenantProfileConfigura
     private long maxStateSizeInKBytes = 32;
     @Schema(example = "2")
     private long maxSingleValueArgumentSizeInKBytes = 2;
-    @Schema(example = "60")
-    private long minAllowedDeduplicationIntervalInSecForCF = 60;
+    @Schema(example = "10")
+    private long minAllowedDeduplicationIntervalInSecForCF = 10;
     @Schema(example = "60")
     private long minAllowedAggregationIntervalInSecForCF = 60;
+    @Schema(example = "300")
+    private long minAllowedIntermediateAggregationIntervalInSecForCF = 300;
```

Comment:
Is it "min allowed"? I think it should just be `intermediateAggregationIntervalInSecForCF`, since it is not configurable on CF level